man/rgbasm.5

```diff
@@ -345,18 +396,21 @@ Some integer operators like
 and
 .Sq -
 don't care whether the operands are integers or fixed-point.
-You can easily truncate a fixed-point number into an integer by shifting it right by 16 bits.
-It follows that you can convert an integer to a fixed-point number by shifting it left.
+You can easily truncate a fixed-point number into an integer by shifting it right by the number of fractional bits.
+It follows that you can convert an integer to a fixed-point number by shifting it left that same amount.
```

Comment:
`* 1.0` also works. Should we mention that?

Reply:
"Shift left/right by N is equivalent to mul/div by 1.0qN" is another example of the kind of fixpoint trivia that belongs in a separate document. This is getting increasingly bloated in tangents trying to teach people what fixpoint is and why it isn't the floatpoint they're used to (not that they're "used to" that either, with "omg 0.1 + 0.2 != 0.3 wtf bbq" being so common).